.github/workflows/test_ert_with_slurm.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+      python-version:
+        type: string
+jobs:
+  test-poly-on-slurm:
+    name: Run ert tests
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
This fetches all branches and all tags, maybe `1` is sufficient?

Reply:
yes, and `1` is even the default.

Reply:
fixed